x-element.js

```diff
@@ -8,7 +8,7 @@ export default class XElement extends HTMLElement {
 
   /** Default templating engine. Use "templateEngine" to override. */
   static get defaultTemplateEngine() {
-    return TemplateEngine.interface;
+    return TemplateEngineWrapper.interface;
```

Comment:
nit: can we eliminate `TemplateEngineWrapper`?

Reply:
I reworked some of this to remove this wrapper. As mentioned, the main point was internal ergonomics around the `repeat` shim. I just created another internal `#mapOrRepeat` function to handle it so that the external interface remains intact.